aten/src/ATen/core/op_registration/op_registration_test.cpp

```diff
@@ -46,50 +46,6 @@ struct MockKernel final : OperatorKernel {
   bool* called_;
 };
 
-TEST(OperatorRegistrationTest, whenRegisteringSameSchemaWithAliasAnalysisAfterRegisteringWithoutAliasAnalysis_thenCanBeCalled) {
```

Comment:
In general, a lot of the tests in this file seemed to test behavior that relies on the fact that you can call RegisterOperators() multiple times for the same op. Since doing so is now an error (doing so causes a duplicate def), I ended up deleting most of those tests.